docs/docs/configuration/masks.md

```diff
@@ -5,81 +5,138 @@ title: Masks
 
 ## Motion masks
 
-Motion masks are used to prevent unwanted types of motion from triggering detection. Try watching the Debug feed (Settings --> Debug) with `Motion Boxes` enabled to see what may be regularly detected as motion. For example, you want to mask out your timestamp, the sky, rooftops, etc. Keep in mind that this mask only prevents motion from being detected and does not prevent objects from being detected if object detection was started due to motion in unmasked areas. Motion is also used during object tracking to refine the object detection area in the next frame. _Over-masking will make it more difficult for objects to be tracked._
+Motion masks tell Frigate where motion should be ignored when deciding whether to run object detection or continue tracking an object.
```

Comment:
> or continue tracking an object.

this is not correct, it does not stop it from tracking objects that have been detected